File: Iniciante/1064.c
#include <stdio.h>
 
int main() {

    int i, nPos=0;
    float n, somaN;

    for(i=0; i < 6; i++){
        scanf("%f", &n);
        if(n > 0){
            somaN = somaN + n;
            nPos++;
        }
    }
        printf("%d valores positivos\n", nPos);
        printf("%.1f\n", somaN / nPos);
    

    return 0;
}

File: Iniciante/1064.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        int nPos=0;
        float somaN=0;

        for(int i=0; i < 6; i++){
            float n = scanf.nextFloat();
            if(n > 0){
                somaN = somaN + n;
                nPos++;
            }
        }
            System.out.printf("%d valores positivos\n", nPos);
            System.out.printf("%.1f\n", somaN / nPos);
    }
}